src/buildstream/_remotespec.py

```diff
@@ -176,6 +181,13 @@ def credentials(self) -> ChannelCredentials:
             )
         return self._credentials
 
+    # grpc keepalive timeout
+    #
+    @property
+    def keepalive_time(self) -> Optional[int]:
+        if self._connection_config:
+            return self._connection_config.get_int("keepalive-timeout", None)
```

Comment:
```suggestion
            return self._connection_config.get_int("keepalive-time", None)
```

Reply:
Should this say keepalive time or timeout? I know the problem I currently have is being inconsistent. https://grpc.io/docs/guides/keepalive/ talks of keepalive timeout.

Reply:
If you look carefully, it talks about both KEEPALIVE_TIME and KEEPALIVE_TIMEOUT (The time between pings vs how long to wait for a response to a ping). 

This is about the keepalive time.